bundles/org.openhab.ui/web/src/assets/definitions/metadata/alexa/helpers.js

```diff
@@ -0,0 +1,63 @@
+import { UNITS_OF_MEASURE } from './constants.js'
+
+export const camelCase = (string, pascalCase = false) =>
+  string
+    .toLowerCase()
+    .split(/[ _-]/)
+    .map((word, index) => (!index && !pascalCase ? word : word.charAt(0).toUpperCase() + word.slice(1)))
+    .join('')
+
+export const titleCase = (string) =>
+  string
+    .toLowerCase()
+    .split(/[ _-]/)
+    .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
+    .join(' ')
+
+export const getGroupParameter = (parameter, groups) => {
+  for (const group of groups) {
+    const config = group.metadata.alexa.config || {}
+    if (parameter in config) return config[parameter]
+  }
+}
+
+export const getOptions = (options, preserve = false) =>
+  Array.isArray(options)
+    ? options.map((value) => ({ value, label: preserve ? value : titleCase(value) }))
+    : Object.keys(options).map((value) => ({ value, label: options[value] }))
+
+export const getSemanticFormat = (type, format) =>
+  Object.keys(format).reduce(
+    (value, key, index, array) =>
+      `${value}${index ? (index === array.length - 1 ? ' or ' : ', ') : ''}` +
+      `<code>${Array.isArray(format) || key === 'default' ? type : camelCase(`${key}_${type}`)}=${format[key]}</code>`,
+    ''
+  )
+
+export const getTemperatureScale = (item) => {
+  const itemType = item.groupType || item.type
+  const state = (item.state !== 'NULL' && item.state !== 'UNDEF' && item.state) || ''
+  const statePresentation = (item.stateDescription && item.stateDescription.pattern) || ''
+  const format = (itemType === 'Number:Temperature' && state) || statePresentation
+  if (format.endsWith('°C')) return 'CELSIUS'
+  if (format.endsWith('°F')) return 'FAHRENHEIT'
+  const { measurementSystem } = (item.settings && item.settings.regional) || {}
+  if (measurementSystem === 'SI') return 'CELSIUS'
+  if (measurementSystem === 'US') return 'FAHRENHEIT'
+}
+
+export const getUnitOfMeasure = (item) => {
+  const itemType = item.groupType || item.type
+  const state = (item.state !== 'NULL' && item.state !== 'UNDEF' && item.state) || ''
+  const statePresentation = (item.stateDescription && item.stateDescription.pattern) || ''
+  const format =
+    ((itemType === 'Dimmer' || itemType === 'Rollershutter') && '%') ||
+    (itemType.startsWith('Number:') && state) ||
+    statePresentation
+  return Object.keys(UNITS_OF_MEASURE).find((id) => format.endsWith(UNITS_OF_MEASURE[id]))
+}
+
+export const link = (title, anchor) => {
+  const link = `https://www.openhab.org/link/alexa#${anchor || title.replace(/[. ]/g, '-').toLowerCase()}`
```

Comment:
Beware that before the next stable version is released, up-to-date documentation is found at https://next.openhab.org, not www.openhab.org.
You can determine if you're running a release build with code like this:
```js
this.$store.state.runtimeInfo.buildString === 'Release Build' ? 'www' : 'next'
```
(but only in the context of a Vue component where `this` is defined), outside that context I'm not sure, maybe importing the store would work?

Reply:
I added support for the different documentation links per your recommendation. For outside that context, I am using a placeholder `%DOC_URL%` that gets replaced once back in context.